projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -50,7 +50,7 @@ describe('BreadcrumbsComponent', () => {
     });
 
     spy = spectator.inject(NavigationService).navigateWithinApp;
-    spy.mockReset();
+    spy.mockRestore();
```

Comment:
Are these two functions equivalent?

Reply:
no they're not

Reply:
-MockReset: Does everything that mockFn.mockClear() does, and also removes any mocked return values or implementations.
-MockRestore: Does everything that mockFn.mockReset() does, and also restores the original (non-mocked) implementation. This is useful when you want to mock functions in certain test cases and restore the original implementation in others. The restoreMocks configuration option is available to restore mocks automatically between tests.